src/components/message/message.component.ts

```diff
@@ -17,6 +17,7 @@ import {
 
 export class MessageComponent {
   @Input() data: ChatbotMessage;
+  @Input() imgText: string;
```

Comment:
I don't see where this is being used?

Reply:
Sorry, removed the code as it was a leftover from before.